denops/slack-status/app.ts

```diff
@@ -37,6 +39,15 @@ main(async ({ vim }) => {
     return;
   }
 
+  statusEmoji = await vim.g.get(
+    "slack_status_emoji",
+    ":speech_balloon:",
+  ) as string;
+
+  if (statusEmoji === "") {
+    statusEmoji = ":speech_balloon:";
+  }
```

Comment:
Since the default value is set, I think it is not necessary to check for an empty string.

Reply:
I thinked When `let g:slack_status_emoji = ""` is written in vimrc, the request `{..., body: {..., status_emoji:""}}` will be invalid and blocked by the API.
But, even if I deleted the code, the code ran successfully.
I'll fix it.